scripts/jest/base.config.js

```diff
@@ -6,7 +6,7 @@
  */
 
 module.exports = {
-    testMatch: ['<rootDir>/**/__tests__/*.spec.(js|ts)'],
+    testMatch: ['<rootDir>/**/__tests__/**/*.spec.(js|ts)'],
```

Comment:
Making the `testMatcher` more flexible to better structure `@lwc/rollup-plugin` tests.